src/main/java/com/pinterest/secor/io/impl/MessagePackSequenceFileReaderWriterFactory.java

```diff
@@ -76,13 +77,17 @@ public KeyValue next() throws IOException {
                 MessageUnpacker unpacker = MessagePack.newDefaultUnpacker(mKey.getBytes());
                 int mapSize = unpacker.unpackMapHeader();
                 long offset = 0;
+                long timestamp = 0;
```

Comment:
Should the default value of timestamp be -1, to be consistent with the KeyValue class?